.changeset/blue-rats-notice.md

```diff
@@ -0,0 +1,10 @@
+---
+"eslint-config-godaddy-react-typescript": major
+"eslint-config-godaddy-typescript": major
+"eslint-config-godaddy-react": major
+"eslint-config-godaddy": major
+---
+
+chore(eslint)!: migrate all eslint configs to eslint@v9
+
+BREAKING CHANGE: removes v8 support and transistions configs to flat config
```

Comment:
Might want to explicitly call out that these are now `type:module` instead of the previous `cjs` format.